_posts/2020-05-13-cockpit-219.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Cockpit 219
+author: kkoukiou
+date: 2020-05-13
+tags: cockpit linux
+slug: cockpit-219
+category: release
+summary: Cockpit with XYZ
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 219.
+
+### Logs: Improved journal logs filtering
+
+It's now possible to filter logs by providing keywords in a text filter which may include the unit of interest, time constraints, priority and [journal fields].
+There is also the possibility to pause/resume the logs streaming.
+Lastly journal logs which appear in other pages are now linked to the the full journal in the Logs page.
+
+![Journal Logs Filtering](/images/journal-logs-filtering.png)
+![Journal Logs Per Page Link](/images/journal-logs-per-page-link.png)
+
+### Administative access improvements
+
+Cockpit now allows more control over your access level. You can gain
+administrative access in a running session, and Cockpit will remember
+that for the next session.
+
+Cockpit respects the "sudo" configuration for your account. For
+example, when NOPASSWD is in your configuration, you don't have to
+provide a password in Cockpit to gain administrative access, and if
+"sudo" is configured for two factor authentication, Cockpit will ask
+twice.
+
+Cockpit only supports "sudo" to elevate the access level. If your
+custom Cockpit page relies on Polkit to grant access, you now need to
+use the superuser option for the relevant channels. For example,
+using the UDisks2 D-Bus API now has to look like this:
+
+`conn = cockpit.dbus("org.freedesktop.UDisks2", { superuser: "try" })`
+
+We plan to bring Polkit support back shortly.
+
+Here is a [demo](https://www.youtube.com/watch?v=-JDMe015m-k) of the new functionality.
```

Comment:
Please embed the video, like we do in other blogs:

    <iframe width="960" height="720" src="https://www.youtube.com/embed/-JDMe015m-k?rel=0" frameborder="0" allowfullscreen></iframe>